### PR TITLE
 #162080971 User can view profile page

### PR DESCRIPTION
**Pull request by mindsworth**
#### What does this PR do?
Implement view user profile page

#### Description of Task to be completed?
- Fixed the header and the notification icon
- A user can view their profile page

#### How should this be manually tested?
Anyone with the right url can view any user profile. From localhost;
1. After cloning or pulling the branch to your local
2. Run yarn install to install all necessary packages and run yarn start:dev start your app on port 5000
3. visit the right url e.g `http://localhost:8080/profile/Prince_41` to see a user profile.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
-[#162080971](https://www.pivotaltracker.com/story/show/162080971)
#### Screenshots (if appropriate)
screenshot 2018-11-21 at 12 39 33 pm

#### Questions:
N/A

### CODE REVIEW

**Review by obulaworld — APPROVED** [review]
Looking Good To Me

File: src/action/__tests__/userProfile.test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// third party library
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import http from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+
+// action types
+import * as types from '../../actionTypes/userProfile';
+
+import {
```
Comment (e0works) [inline src/action/__tests__/userProfile.test.js:11]: add comment to this import statement

File: src/components/profile/profileHeader/ProfileHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// react libraries
+import React, { Component } from 'react';
+
+// third party libraries
+import PropTypes from 'prop-types';
+
+/**
+ *
```
Comment (e0works) [inline src/components/profile/profileHeader/ProfileHeader.jsx]: remove this empty line on jsdoc

File: src/components/profile/profileSidebar/FollowLogCard.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```
Comment (e0works) [inline src/components/profile/profileSidebar/FollowLogCard.jsx:2]: add a comment to this import statement

File: src/components/profile/profileSidebar/profileEdit/ProfileEdit.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
```
Comment (e0works) [inline src/components/profile/profileSidebar/profileEdit/ProfileEdit.jsx:2]: add a comment to this import statement

File: src/containers/profile/__test__/profile.test.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+// React Libraries
+import React from 'react';
```
Comment (e0works) [inline src/containers/profile/__test__/profile.test.js:2]: `react-redux` is not part of `React` libraries
it should be under, third-party libraries

**Review by e0works — CHANGES_REQUESTED** [review]

File: src/action/__tests__/userProfile.test.js (hunk 1 above)
Comment (mindsworth) [reply src/action/__tests__/userProfile.test.js:11]: done

File: src/components/profile/profileHeader/ProfileHeader.jsx (hunk 2 above)
Comment (mindsworth) [reply src/components/profile/profileHeader/ProfileHeader.jsx]: done

File: src/components/profile/profileSidebar/profileEdit/ProfileEdit.jsx (hunk 4 above)
Comment (mindsworth) [reply src/components/profile/profileSidebar/profileEdit/ProfileEdit.jsx:2]: done

File: src/containers/profile/__test__/profile.test.js (hunk 5 above)
Comment (mindsworth) [reply src/containers/profile/__test__/profile.test.js:2]: done

**Review by e0works — APPROVED** [review]

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**